### PR TITLE
Feature/landing page

### LINKED ISSUE #22: As a developer I want to have a nicely looking landing page
Like Objects API.

### PR DESCRIPTION
**Pull request by annashamray**
fixes https://github.com/maykinmedia/sharing-configs/issues/22

### CODE REVIEW

File: src/sharing/scss/_app.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+h1,
+h2,
+h3,
+h4,
+h5 {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-weight: 500;
```
Comment (joeribekker) [inline src/sharing/scss/_app.scss:8]: Can we make the font-weight for h1=300?

File: src/sharing/scss/_app.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  line-height: 1.2;
+}
+
+p {
+  margin-top: 0;
+  margin-bottom: 1rem;
+}
+
+body {
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```
Comment (joeribekker) [inline src/sharing/scss/_app.scss:18]: Can we include a Google font, Roboto, to stay close to what it is now?

```suggestion
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

**Review by joeribekker — CHANGES_REQUESTED** [review]
2 minor requests, otherwise looks good and I think it's BEM-ish :)

File: src/sharing/scss/_app.scss (hunk 1 above)
Comment (annashamray) [reply src/sharing/scss/_app.scss]: Done, but I don't see the difference :)

**Comment by joeribekker** [comment]
> Done, but I don't see the difference :)

That's because you didn't implement the Roboto font, you only said to CSS to use it. Please also add the Roboto font code.

**Review by joeribekker — APPROVED** [review]

**Pull request merged**